src/render/OpenGL.hpp

```diff
@@ -92,21 +113,8 @@ struct SMonitorRenderData {
     bool         blurFBShouldRender = false;
 
     // Shaders
-    bool    m_bShadersInitialized = false;
-    CShader m_shQUAD;
-    CShader m_shRGBA;
-    CShader m_shPASSTHRURGBA;
-    CShader m_shMATTE;
-    CShader m_shRGBX;
-    CShader m_shEXT;
-    CShader m_shBLUR1;
-    CShader m_shBLUR2;
-    CShader m_shBLURPREPARE;
-    CShader m_shBLURFINISH;
-    CShader m_shSHADOW;
-    CShader m_shBORDER1;
-    CShader m_shGLITCH;
-    CShader m_shCM;
+    bool                 m_bShadersInitialized = false;
+    SP<SPreparedShaders> m_shaders;
```

Comment:
while we're at it, `m_shaders` should probably be in CHyprOpenGL. I don't think there is a reason to split shaders between monitors.